eng/.docsettings.yml

```diff
@@ -52,6 +52,7 @@ known_content_issues:
   - ['sdk/keyvault/Microsoft.Azure.KeyVault/README.md','#5499']
   - ['sdk/servicebus/Microsoft.Azure.ServiceBus/README.md','#5499']
   - ['sdk/eventhub/Microsoft.Azure.EventHubs/README.md','#5499']
+  - ['sdk/eventhub/Azure.Messaging.EventHubs/README.md','#5499']
```

Comment:
small nit, should this be eventhubs rather than eventhub?

Reply:
Yes, it should, but the path had to match the repository for the REST specs, unfortunately.